### DISCUSSION TITLE
If I delete file , will it redownload ?

### DISCUSSION THREAD

**Discussion by Emilien-Etadam** [Dineshkarthik/telegram_media_downloader#104] [Q&A]
Can I delete the file without the script re-downloading it? 
I need to specify that the "last_read_message_id:" didn't change since I moved it on my Synology NAS...

**Comment by Dineshkarthik** [comment]
If you delete the downloaded files and didn't change anything in the `config.yaml` it won't re-download.

**Reply by Emilien-Etadam** [reply]
ok but config.yaml is not incrementing since I run the script from the synology nas.

**Comment by Emilien-Etadam** [comment]
If I put "last_read_message_id: " to "0"  it will re-download everything or just download missing files ?
